### PR TITLE
Testrunner-Toolkit docs eOL

### PR DESCRIPTION
**Pull request by sweeneyskirt-sl**
Updated links to new docs

### CODE REVIEW

**Comment by tianfeng92** [comment]
LGTM. But it may need rebasing master and updating this example as well https://github.com/saucelabs/saucectl-cypress-example/tree/master/examples/allure-plugin

**Comment by sweeneyskirt-sl** [comment]
Thanks, @tianfeng92 I think it should be all set now.

File: examples/typescript/.sauce/config.yml [hunk 1]
```diff
@@ -16,7 +16,15 @@ docker:
   # Affects how test files are transferred to the docker container when using the docker mode.
   fileTransfer: copy # Choose between mount|copy.
 cypress:
+<<<<<<< HEAD
+<<<<<<< HEAD
```
Comment (tianfeng92) [inline examples/typescript/.sauce/config.yml:20]: should be fixed here:)

Comment (sweeneyskirt-sl) [reply examples/typescript/.sauce/config.yml:20]: Grrr. I feel like I've fixed that 3 times! Thanks, @tianfeng92

File: examples/typescript/.sauce/config.yml [hunk 2]
```diff
@@ -16,7 +16,15 @@ docker:
...
+  version: 8.3.0 # See https://docs.saucelabs.com/dev/cli/saucectl/#supported-frameworks-and-browsers for a list of supported versions.
+=======
   version: 9.3.1 # See https://docs.saucelabs.com/testrunner-toolkit for a list of supported versions.
+>>>>>>> d9a4f81ca203197b0c28d2fdd5d1645a2f6dabd7
+=======
+  version: 9.3.1 # See https://docs.saucelabs.com/dev/cli/saucectl/#supported-frameworks-and-browsers for a list of supported versions.
+>>>>>>> 0af4fe9276c97c714797413f43588ff189f565e4
```
Comment (sweeneyskirt-sl) [inline examples/typescript/.sauce/config.yml:27]:
```suggestion
  version: 9.3.1 # See https://docs.saucelabs.com/dev/cli/saucectl/#supported-frameworks-and-browsers for a list of supported versions.
```

**Review by alexplischke — APPROVED** [review]

**Pull request merged**